R/example.R

```diff
@@ -435,42 +440,53 @@ example_jira_issue_components <- function(folder_path="/tmp",folder_name) {
 #' @return the JSON path of the newly created issue issue tracker
 #' @export
 #' @keywords internal
-example_jira_two_issues <- function(folder_path="/tmp",folder_name) {
+example_jira_two_issues <- function(folder_path, folder_name) {
 
   # Create folder & repo
-  folder_path <- io_make_folder(folder_path=folder_path, folder_name = folder_name)
-
-  issue1 <- make_jira_issue(jira_domain_url = "https://project.org/jira",
-                            issue_key = "GERONIMO-123",
-                            issue_type = "New Feature",
-                            status = "Open",
-                            resolution = "Finished",
-                            title = "This is a summary.",
-                            description = "The new features have been implemented.",
-                            components = "x-core",
-                            creator_name = "Bob",
-                            reporter_name = "Joe",
-                            assignee_name = "Moe",
+  folder_path <- io_make_folder(folder_path = folder_path, folder_name = folder_name)
+
+  issue1 <- make_jira_issue(
+    jira_domain_url = "https://project.org/jira",
+    issue_key = "PROJECT-11",
+    project_key = "PROJECT",
+    summary = "Summary of issue 1",
+    description = "Description of summary 1",
+    issue_type = "New Feature",
+    resolution = "Finished",
+    priority = "Minor",
+    status = "Closed",
+    labels = c("pull-request-available"),
+    components = c("jira"),
+    affects_versions = c("1.1.1"),
+    fix_versions = c("1.1.1"),
+    assignee_name = "Moe",
+    creator_name = "Bob",
+    reporter_name = "Joe"
   )
 
-  issue2 <- make_jira_issue(jira_domain_url = "https://project.org/jira",
-                            issue_key = "GERONIMO-124",
-                            issue_type = "New Feature",
-                            status = "Open",
-                            resolution = "Finished",
-                            title = "This is a summary.",
-                            description = "The new features have been implemented.",
-                            components = "x-spring",
-                            creator_name = "Moe",
-                            reporter_name = "Larry",
-                            assignee_name = "Curly",
+  issue2 <- make_jira_issue(
+    jira_domain_url = "https://project.org/jira",
+    issue_key = "PROJECT-22",
+    project_key = "PROJECT",
+    summary = "Summary of issue 2",
+    description = "Description of summary 2",
+    issue_type = "New Feature",
+    resolution = "Finished",
+    priority = "Minor",
+    status = "Open",
+    labels = c("pull-request-available"),
+    components = c("jira"),
+    affects_versions = c("2.2.2"),
+    fix_versions = c("2.2.2"),
+    assignee_name = "Steven",
+    creator_name = "Nathan",
```

Comment:
To double check, these are random names and a random release right?

Reply:
Yes, those are random names and versions